Query Fix and file name change

oladele-david opened the pull request:
Changed filename of the loacation.php to => location.php
fixed the 
        $query2 = "select state.*,country.cntry_name from state,country order by id desc";
to
        $query2 = "SELECT s.*, c.cntry_name FROM state AS s JOIN country AS c ON s.c_id = c.id ORDER BY s.id DESC;";

the query was giving the wrong values

detronetdip commented on Admin/locationpin.php:
@oladele-david can you please provide me with suitable examples that what is expected result and what it is giving wrong? It will be helpful.

detronetdip reviewed (changes requested):
I have utilized the Indian currency (Rupee) in this project. However, I'd like to point out that making a pull request to change the currency to your local currency might not be feasible, as the project's users come from various countries with different currencies. Therefore, it would be best to maintain the original commit. Your understanding of this situation is greatly appreciated.

detronetdip reviewed (commented):
LGTM

detronetdip commented:
@oladele-david please do the necessary changes that I have already mentioned above and once done let me know I will me more than happy to merge it

oladele-david commented:
I have made the necessary revision required. I reverted the initial  pull request and fixed the bug i noticed in the **Loacation.php**  file in the admin folder.


$query2 = "select state.*,country.cntry_name from state,country order by id desc";
which load incorrectly as shown in the image below:

![Screenshot from 2023-08-15 03-03-39](https://github.com/detronetdip/E-commerce/assets/46281101/b707db9c-1ef8-4123-a2bd-6e380e8a1752)

It returns all the countries and also 1 state in the loop


which i rewrote to this 
$query2 = "SELECT s.*, c.cntry_name FROM state AS s JOIN country AS c ON s.c_id = c.id ORDER BY s.id DESC;";

![Screenshot from 2023-08-15 03-04-04](https://github.com/detronetdip/E-commerce/assets/46281101/5adb0fa8-3aa8-43f8-9ad2-02b85bb7c2f6)

which behaves as expected. 

That was the only changes made. Please this is my first PR :smile:  so bear with me :pray:

detronetdip commented:
LGTM
